packages/react/src/CTAForm/CTAForm.module.css

```diff
@@ -0,0 +1,47 @@
+.CTAForm {
+  display: grid;
+  gap: var(--base-size-16);
+  grid-template-columns: 2fr auto;
+  align-items: flex-end;
+}
+
+.CTAForm > * {
```

Comment:
Not sure if things have changed now, but I recall wildcard selectors not being recommended. Can we do this with comma separated class names instead?

Reply:
I hear what you are saying and I would agree that wildcard selectors could be dangerous in some cases, but for this I believe that they would be both safe and effective so that users could include their own children without issue. What this specific declaration is doing is setting all children to span the full size of the grid. When a user adds and additional input that would be added with this css and the size would be as they might expect it to be. Does that make sense and what would your recommendation be?

Reply:
Thanks for clarifying, I can see how custom css classes won't serve the use-case for unknown children.

I can't think of another option besides using the ~ sibling selector for tighter specificity, which I'm guessing you've already considering. It ultimately would achieve the same, so happy to also stick with this 👍